aip/aog/3010.md

```diff
@@ -16,27 +16,31 @@ permalink: /aog/3010
 # Actions on Google Vertical Integration Webhook Format
 
 If an action is part of a vertical program that requires a webhook for Google
-to integrate with the action, the webhookd format **must** follow this AIP.
+to integrate with the action, the webhook format **must** follow this AIP.
 
 ## Guidance
 
 ### Custom method name
 
 Custom methods in a vertical program are defined by Google and implemented by
-third-party actions. If word separation is required for the method name,
-`lowerCamelCase` **should** be used.
+third-party actions. Guidelines related to names of custom methods in
+[AIP-136](../0136.md) **should** be applied.
+
+- The name of the method **should** be a verb followed by a noun.
```

Comment:
Should this also include the subpoint "The name must not contain prepositions (“for”, “with”, etc.)."?

Reply:
added